### DISCUSSION TITLE
Configure Renovate PR wants to update packages I don't have

### DISCUSSION THREAD

**Discussion by Avasam** [renovatebot/renovate#8182] [Request Help] [answered]
I am trying to configure Renovate for the first time. Here is the PR in question: https://github.com/Avasam/speedrun.com_global_scoreboard_webapp/pull/162
I'm a bit confused as to why Renovate wants to updates certain packages, (like grunt, karma, mocha, codecov, etc.), as I don't use them and they are nowhere to be found in package.json or package-lock.json.

**Comment by viceice** [comment, marked as answer]
Did you checked all package.json files listed in the onboarding pr?

**Reply by viceice** [reply]
https://www.github.com/Avasam/speedrun.com_global_scoreboard_webapp/tree/develop/tournament-scheduler%2Fdependencies%2Freact-add-to-calendar%2Fpackage.json

You've configured base branches, so renovate will only check them

**Reply by viceice** [reply]
You maybe should exclude the dependency folders if they are updated externally

**Reply by Avasam** [reply]
oh yeah, the dependencies folder is manually updated and should absolutely be excluded. I thought I had checked whether the listed libraries came from there. Figured I had missed something simple. Thanks!
